packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -192,6 +193,11 @@ export interface TableOptions {
 
   /**
    * The name of TTL attribute.
+   *
+   * If a value is passed a custom resource is created. The custom resource does add support
+   * for changes to the TTL attribute later.
+   *
+   * If you wish to remove the custom resource and disable the TTL, you can leave the default (undefined).
```

Comment:
Let's add a safety valve here, just in case. A property that disables creation of the custom resource. Something like `timeToLiveCustomResource?: boolean`, which is `true` by default. Let's include the fact that you can turn this off in the ReadMe.

This way, you can set this in the integration tests, and you can avoid changing them! (Do leave it on in one of the integ tests though, we should have this functionality covered by them)